### PR TITLE
Add inital API contract

### PR DESCRIPTION
**Pull request by DougWelchons**
# Adds
- APIContract: added the initial api endpoints and formatting

### CODE REVIEW

File: APIContract.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        "good with other pets": true,
+        "photo_url_1": "URL Path",
+        "photo_url_2": "URL Path",
+        "photo_url_3": "URL Path",
+      }
+    }
+  }
+  ```
+
+## Delete a Booking
```
Comment (arikalea) [inline APIContract.md]: Need to change the wording on this one from 'booking' to 'pet'

**Review by arikalea — APPROVED** [review]
This looks really great! This will make it so mush easier to add the endpoint info to the readme.

File: APIContract.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+# Yardsourcing API Contract
```
Comment (arikalea) [inline APIContract.md]: Change this title to Fetching Friends

File: APIContract.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+## Endpoints
+
+| HTTP verbs | Paths  | Used for | Output |
+| ---------- | ------ | -------- | ------:|
+| GET | /api/v1/pets/:type | Get all pets of a given type | [json](#pets-by-type) |
+| GET | /api/v1/users/:user_id | Get a user, and users pets, favorited  pets, and applications| [json](#all-user-data) |
+| POST | /api/v1/users | Create a new user | [json](#create-user) |
+| POST | /api/v1/pets | Create a new pet | [json](#create-pet) |
+| DELETE | /api/v1/pets/:pet_id  | Delete a pet | [details](#delete-a-yard) |
+| PUT | /api/v1/yards/:yard_id | Update a yard | [json](#update-a-yard)
```
Comment (arikalea) [inline APIContract.md]: Change 'yards' here. I'm assuming change it to 'pet'?

**Review by arikalea — APPROVED** [review]
We just need to update a few of the names, otherwise looks good!

File: APIContract.md (hunk 2 above)
Comment (DougWelchons) [reply APIContract.md]: good catch!

File: APIContract.md (hunk 3 above)
Comment (DougWelchons) [reply APIContract.md]: Thanks I'll fix that.

File: APIContract.md (hunk 1 above)
Comment (DougWelchons) [reply APIContract.md]: Done! thanks

**Review by arikalea — APPROVED** [review]

**Pull request merged**